packages/core/test/hotkeys/hotkeysTests.tsx

```diff
@@ -114,19 +114,32 @@ describe("Hotkeys", () => {
         });
 
         it("triggers hotkey dialog with \"?\"", (done) => {
+            // this test takes awhile. the hotkey dialog is supposed to take 100ms
```

Comment:
@cmslewis please change to `it.skip(...)` to ignore this test and file an issue to track refactoring it.